### DISCUSSION TITLE
2023 how to use flash

### DISCUSSION THREAD

**Discussion by 839781960** [cefsharp/CefSharp#4461] [General]
2023 how use flash
which version？

**Comment by amaitland** [comment]
In future please use [Discussions](https://github.com/cefsharp/CefSharp/discussions) to ask support related questions.

Flash support was removed a number of years ago. It's not practical to provide support for versions that old.

A quick search yields https://github.com/Mzying2001/CefFlashBrowser

You'll have to do your own research if you have further questions.
